src/utils/utils.ts

```diff
@@ -31,3 +33,26 @@ export async function executeCommand(command: string, cwd?: string): Promise<str
 export function generateQuotedFullName(schema: string, objectName: string): string {
 	return `[${escapeClosingBrackets(schema)}].[${escapeClosingBrackets(objectName)}]`;
 }
+
+
+/**
+ * Copied from Azure function: Create Function
```

Comment:
For something as basic as this I don't think it matters to link the source. Instead it would be more helpful to have the description actually say what this is doing